sdk-scheme-adapter/v2_1_0/components/schemas/accountDeletionResponse.yaml

```diff
@@ -0,0 +1,28 @@
+type: object
+required:
+  - accountIdType
+  - accountIdValue
+properties:
+  accountIdType:
+    $ref: ./PartyIdType.yaml
+  accountIdValue:
+    $ref: ./PartyIdentifier.yaml
+  accountSubIdOrType:
+    $ref: ./PartySubIdOrType.yaml
```

Comment:
Can we change these to idType, idValue and idSubValue just to align with the POST /accounts response?